src/env_loader.zig

```diff
@@ -149,7 +150,7 @@ pub const Loader = struct {
                 var no_proxy_list = std.mem.split(u8, no_proxy_text, ",");
                 var next = no_proxy_list.next();
                 while (next != null) {
-                    var host = next.?;
+                    var host = std.mem.trim(u8, next.?, &std.ascii.whitespace);
```

Comment:
instead of using `std.ascii.whitespace` and `std.mem.trim`, can you use `strings.trim` with the whitespace characters from `lengthOfLeadingWhitespaceASCII`? probably best to move them into a pub const in `strings_immutable.zig`

Reply:
I have replaced `std.mem.trim` with `strings.trim`.

I have also tried to extract the constant, and found out that the whitespace characters from `lengthOfLeadingWhitespaceASCII` are identical to `std.ascii.whitespace`. `lengthOfLeadingWhitespaceASCII` uses its own set because it spells them out in `case`.

So this will create a constant that's the same as one in `std.ascii` and `lengthOfLeadingWhitespaceASCII` won't be able to reuse it. Should I do it, or is it OK to keep the code as is?

Reply:
> Here's a little comptime magic to spread the items from whitespace_chars into the switch statement.

Thanks, this is neat. And the assembly is pretty similar as well.

> Sorry this is annoying, but we want to make sure this array of whitespace chars doesn't change when upgrading zig

Not annoyng at all. I have forgotten that Zig does not guarantee stability of its stdlib yet.

There are several uses of `std.ascii.is{Whitespace,Control,Alphanumeric}` functions in the codebase, should they also be pulled into Bun proper?

Reply:
for now this is great